ql/termstructures/yield/ratehelpers.hpp

```diff
@@ -347,6 +358,7 @@ namespace QuantLib {
         Handle<YieldTermStructure> discountHandle_;
         RelinkableHandle<YieldTermStructure> discountRelinkableHandle_;
         ext::optional<bool> useIndexedCoupons_;
+        bool swapBuiltViaGivenBuilder;
```

Comment:
I believe this issue "_'SwapRateHelper::swapBuiltViaGivenBuilder' is never used_" from "Codacy Static Code Analysis" is a false positive. The member is used, e.g. in ratehelpers.cpp:l.681 `SwapRateHelper::setTermStructure()`.